next.config.js

```diff
@@ -1,10 +1,10 @@
-const withMakeswiftInitializer = require("@makeswift/runtime/next/plugin");
+const withMakeswiftInitializer = require('@makeswift/runtime/next/plugin')
 
-const withMakeswift = withMakeswiftInitializer();
+const withMakeswift = withMakeswiftInitializer()
 
 module.exports = withMakeswift({
   reactStrictMode: true,
   images: {
-    domains: ["s.mkswft.com"],
+    domains: ['s.mkswft.com', 'cdn.sanity.io'],
```

Comment:
You shouldn't have to provide `s.mkswft.com` as it should be included by default with the Makeswift Next.js plugin